js/src/hooks/useMCProductStatistics.js

```diff
@@ -27,6 +28,16 @@ const useMCProductStatistics = () => {
 		hasFinishedResolution && data?.loading ? true : false;
 	const hasStats = hasFinishedResolution && data?.statistics ? true : false;
 
+	const [ refreshProductStats ] = useApiFetchCallback( {
+		path: `/wc/gla/mc/product-statistics/refresh`,
+		method: 'GET',
+	} );
+
+	const refreshStats = async () => {
+		await refreshProductStats();
+		invalidateResolution();
+	};
```

Comment:
Maybe we could wrap this function by `useCallback`, as there are dependencies used in the function. But I'm not sure if it is a best practice tho.

Reply:
Thanks @ianlin for the suggestion, I believe it won't make much difference in terms of performance, but it won't hurt either, so I've added it here: https://github.com/woocommerce/google-listings-and-ads/pull/2324/commits/345d019019e423eb7df00d3487e1d77b9c74f137